src/types/index.ts

```diff
@@ -60,3 +60,11 @@ export type Step = {
   tooltip: string;
   state: StepState;
 };
+
+export type Address = string;
+export type QueryArgs = Record<string, any>;
+export type QueryAttrs = Record<string, any>;
+export type QueryBuilder = (
+  args?: QueryArgs,
+  attrs?: QueryAttrs
+) => Record<string, any>;
```

Comment:
@garethfuller is this a good place for them? Right now only the aave one uses it, if you are happy with it I can change the balancer one too.

Reply:
Sure, yeah fine to add them here.